gtars/src/uniwig/mod.rs

```diff
@@ -430,10 +552,98 @@ pub fn uniwig_main(
                 }
             }
         }
+        }
+    );
+
+    let vec_strings = vec!["start", "core", "end"];
+
+    match output_type {
+        "wig" => {
+            println!("Combining Wig Files");
+
+            for location in vec_strings.iter(){
+
+                write_combined_wig_files(*location, output_type, bwfileheader, &chromosomes);
+
+            }
+
+
+
+        }
+        _ => {
+
+        }
+
```

Comment:
Is this a lone `match`? If so, maybe we can improve readability by just calling an `if`